network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/blog_set_block.html

```diff
@@ -0,0 +1 @@
+{% extends "./recent_blog_entries.html" %}
```

Comment:
straight up pass-through: the context variables are all the same, so it will (at least, it should =D) render identically.